homeassistant/components/icloud/translations/de.json

```diff
@@ -28,7 +28,8 @@
                 "data": {
                     "password": "Passwort",
                     "username": "Email",
-                    "with_family": "Mit Familie"
+                    "with_family": "Mit Familie",
```

Comment:
The translations will be done via Lokalize, don't do them here.

Reply:
Okay! Probably should have read up on the documentation a bit more, but it just seemed too tempting :)
Will remove it 👍